jetty-util/src/main/java/org/eclipse/jetty/util/log/JavaUtilLog.java

```diff
@@ -65,8 +65,8 @@
 {
     private static final String THIS_CLASS = JavaUtilLog.class.getName();
     private static final boolean __source =
```

Comment:
Should not this be `SOURCE` since it's a `static final` constant?

Reply:
done.